src/collections/members/sandra-ashipala/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Sandra Ashipala
+position: Contributor
+image_path: ./sandra-ashipala.webp
+github: sandramsc
+twitter: sandraamsc
+linkedin: sandraashipala
+layer5: aaeb3b36-ac02-450a-bb7e-964ad9d818e4
+location: Berlin, Germany
```

Comment:
Relocated back to Namibia from Germany, please update as current location.
```suggestion
location: Namibia
```

Reply:
hey @sandramsc , I have done the required changes suggested from you. Please have a look